src/components/layout/footer/Footer.tsx

```diff
@@ -6,12 +6,13 @@ const footerItems = [
   { title: "Privacy Policy", path: "/privacy" },
 ];
 
-
 const Footer = () => {
   return (
     <Wrapper>
       {footerItems.map(({ title, path }, i) => (
-        <FooterLink to={path}>{title}</FooterLink>
+        <FooterLink key={i} to={path}>
+          {title}
+        </FooterLink>
```

Comment:
Good Change!
Always use key props when you rendering elements from an array (like here sith map function)